applications/tari_console_wallet/src/automation/commands.rs

```diff
@@ -139,6 +140,16 @@ pub async fn burn_tari(
         .map_err(CommandError::TransactionServiceError)
 }
 
+pub async fn create_key_combo<TBackend: KeyManagerBackend + 'static>(
```

Comment:
A better name for this feature is "create key pair"

Reply:
Definitely agreed, I changed it accordingly.